docs/site/tutorials/authentication/Authentication-Tutorial.md

```diff
@@ -209,6 +207,9 @@ export class ShoppingApplication extends BootMixin(
     this.component(AuthenticationComponent);
 
     // ...
+  }
+  // ...
+}
```

Comment:
Before this change, GitHub's syntax highlighter was confused - notice the text in "Securing and Enpoint..." is rendered in blue:

<img width="711" alt="Screen Shot 2019-11-25 at 09 36 02" src="https://user-images.githubusercontent.com/1140553/69524913-5cba9000-0f67-11ea-89db-b54f0c51c103.png">

Also Prettier was not able to automatically format the code. Once I added closing curly braces, Prettier shortened the `import` line and GitHub started to render the text in the next section as regular text (in black color) again.